crates/astria-composer/src/searcher/mod.rs

```diff
@@ -266,7 +265,7 @@ impl Searcher {
 }
 
 fn reconnect_exited_collector(
```

Comment:
`reconnect_exited_geth_collector` maybe?

Reply:
I will rename it